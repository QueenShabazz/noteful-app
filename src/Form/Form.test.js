import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Form from './Form'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() });

describe(`Form component`, () => {

  it('renders the Form default', () => {
    const wrapper = shallow(
        <Form />
        )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
  
})