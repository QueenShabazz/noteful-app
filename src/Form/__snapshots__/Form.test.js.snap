// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component renders the Form default 1`] = `
<form
  action="#"
/>
`;
